features/support/request.js

```diff
@@ -40,8 +40,8 @@ export async function waitForRequest(
     method,
     path,
     matcher,
-    step = 100,
     milliseconds = 1000,
+    step = 100,
```

Comment:
I've moved these parameters around because you are more likely to want to change the milliseconds than you are the step